pkg/sync/v6/data/ConfigMap/openshift-azure-branding/branding.yaml

```diff
@@ -10,6 +10,43 @@ data:
     \   width: 310px;\r\n  }\r\n}\r\n"
+  branding.js: |
+    'use strict';
+
+    (function() {
+        /* Customize the CLI download locations */
+        window.OPENSHIFT_CONSTANTS.CLI = {
+            "Linux (64 bits)": "https://mirror.openshift.com/pub/openshift-v3/clients/3.11.98/linux/oc.tar.gz",
+            "Windows":         "https://mirror.openshift.com/pub/openshift-v3/clients/3.11.98/windows/oc.zip",
+            "Mac OS X":        "https://mirror.openshift.com/pub/openshift-v3/clients/3.11.98/macosx/oc.tar.gz"
+        };
+
+        /* Customize the base URL for all documentation links across the console */
+        window.OPENSHIFT_CONSTANTS.HELP_BASE_URL = "https://docs.openshift.com/aro/";
+
+        /* Customize the CATALOG_HELP_RESOURCES constant */
+        window.OPENSHIFT_CONSTANTS.CATALOG_HELP_RESOURCES.links = [
+            {
+                title: 'Documentation',
+                href: 'https://docs.openshift.com/aro/welcome/index.html'
+            },
+            {
+                title: 'Interactive Learning Portal',
+                href: 'https://learn.openshift.com'
+            },
+            {
+                title: 'Container Development Kit',
+                href: 'https://developers.redhat.com/products/cdk/overview/'
+            },
+            {
+                title: 'YouTube',
```

Comment:
Worth checking with @patrickstrick what content we want to see associated with these dropdowns. implementation lgtm.
OC +
Documents - might want to check if we want MSFT links instead
CDK - need to confirm
Blog - Need to confirm
Youtube - I would remove it. But need to confirm.

Reply:
sounds good to me. We can customize this section (right-hand-side panel) of the console with whatever pieces of information we wish. So we might add a link to the Openshift ARO documentation as well to the Azure ARO documentation and maybe even to the Azure portal. 

I agree that a link to the youtube channel isn't particularly important however all of these, except the Documentation link, are lifted from the default console installation.

@patrickstrick which links do you recommend?

Reply:
It would be good to include the MS doc link as well just after our doc link. The URL is https://docs.microsoft.com/en-us/azure/openshift/

Reply:
Otherwise, it looks great!

Reply:
@patrickstrick do you think CDK, Blog and Youtube links should stay or go?

Reply:
Let's keep them. They are good resources for someone looking to learn as they get started.